gnome-extra/cinnamon-translations/cinnamon-translations-4.8.3.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PLOCALES="af am an ar as ast az be be@latin ber bg bn bn_IN br bs ca ca@valencia crh cs csb cy da de dz el en@shaw en_AU en_CA en_GB en_IE en_NZ en_ZA eo es es_AR et eu fa fi fil fo fr fr_CA frp fur fy ga gd gl gu ha he hi hr hu hy ia id ie ig is it ja jv ka kab kk km kn ko ksw ku ky la li lo lt lv mai mg mi mk ml mn mr ms my nap nb nds ne nl nn no nso oc om or pa pap pl ps pt pt_BR ro ru rue rw sa sc sco shn si sk sl so sq sr sr@ijekavianlatin sr@latin sv sw szl ta te tg th tk tl tlh tpi tr ts tt ug uk ur uz uz@cyrillic vi wa xh yi yo zgh zh_CN zh_HK zh_TW zu"
+
+inherit l10n
```

Comment:
These days we install all localization files unconditionally, but up to you whether you want to update it here or not. But you'll be required to update this at some point. 

https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301

Reply:
That policy seems a little unclear to me. It seems like its targeted at packages that take a long time to rebuild (i.e. rust, libreoffice, gcc, boost), and small files (which I take to be both small in individual size and in total count).

It seems like localization packages (i.e. libreoffice-l10n, cinnamon-transaltions) wouldn't exactly fit that definition, as they are quick to build and would install hundreds of files that take 10s of MBs.